Misc/NEWS.d/next/Library/2023-03-13-12-05-55.gh-issue-102615.NcA_ZL.rst

```diff
@@ -0,0 +1 @@
+Use list instead of tuple in ``repr`` of generic aliases with ``ParamSpec``.
```

Comment:
I think this would be a pretty confusing news entry to people who don't know much about typing/PEP-612:

```suggestion
Typing: Improve the ``repr`` of generic aliases for classes generic over a
:class:`~typing.ParamSpec`. (Use square brackets to represent a parameter
list.)
```